contrib/objectstore/subscriber/objectstore.go

```diff
@@ -181,6 +181,8 @@ func (s *Subscriber) flushFlowsToObject(tag Tag, endTime time.Time) error {
 	endTimeString := strconv.FormatInt(endTime.UTC().UnixNano()/int64(time.Millisecond), 10)
 
 	metadata := map[string]*string{
+		"first-timestamp": aws.String(startTimeString),
+		"last-timestamp":  aws.String(endTimeString),
 		"start-timestamp": aws.String(startTimeString),
```

Comment:
how about removing start and end timestamp fields? just for the name, are we keeping the times twice?

Reply:
I will remove start-timestamp, end-timestamp